Specs/Renderer/CubeMapSpec.js

```diff
@@ -396,6 +481,94 @@ defineSuite([
         var negativeZFloats = [14541, 12902, 13926, 15360];
 
+        var positiveXColor = new Color(0.2, 0.4, 0.6, 1.0);
+        var positiveYColor = new Color(0.6, 0.4, 0.2, 1.0);
+
+        cubeMap = new CubeMap({
+            context : context,
+            source : {
+                positiveX : {
+                    width : 1,
+                    height : 1,
+                    arrayBufferView : new Uint16Array(positiveXFloats)
+                },
+                negativeX : {
+                    width : 1,
+                    height : 1,
+                    arrayBufferView : new Uint16Array(negativeXFloats)
+                },
+                positiveY : {
+                    width : 1,
+                    height : 1,
+                    arrayBufferView : new Uint16Array(positiveYFloats)
+                },
+                negativeY : {
+                    width : 1,
+                    height : 1,
+                    arrayBufferView : new Uint16Array(negativeYFloats)
+                },
+                positiveZ : {
+                    width : 1,
+                    height : 1,
+                    arrayBufferView : new Uint16Array(positiveZFloats)
+                },
+                negativeZ : {
+                    width : 1,
+                    height : 1,
+                    arrayBufferView : new Uint16Array(negativeZFloats)
+                }
+            },
+            pixelDatatype : PixelDatatype.HALF_FLOAT,
+            sampler : new Sampler({
+                wrapS : TextureWrap.CLAMP_TO_EDGE,
+                wrapT : TextureWrap.CLAMP_TO_EDGE,
+                minificationFilter : TextureMinificationFilter.LINEAR,
+                magnificationFilter : TextureMagnificationFilter.LINEAR
+            })
+        });
+
+        var fs =
+            'uniform samplerCube u_texture;' +
+            'void main() { gl_FragColor = textureCube(u_texture, normalize(vec3(1.0, 1.0, 0.0))); }';
+
+        var uniformMap = {
+            u_texture : function() {
+                return cubeMap;
+            }
+        };
+
+        if (!context.textureFloatLinear) {
+            expect({
+                context : context,
+                fragmentShader : fs,
+                uniformMap : uniformMap,
+                epsilon : 1
+            }).contextToRender(positiveYColor.toBytes());
+        } else {
+            Color.multiplyByScalar(positiveXColor, 1.0 - 0.5, positiveXColor);
+            Color.multiplyByScalar(positiveYColor, 0.5, positiveYColor);
+            var color = Color.add(positiveXColor, positiveYColor, positiveXColor);
+            expect({
+                context : context,
+                fragmentShader : fs,
+                uniformMap : uniformMap,
+                epsilon : 1
+            }).contextToRender(color.toBytes());
+        }
+    });
+
+    it('creates a cube map with half floating-point textures and linear filtering', function() {
```

Comment:
It looks like `creates a cube map with half floating-point textures` and `creates a cube map with half floating-point textures and linear filtering` are actually testing the opposite thing, should the titles be swapped?